### ISSUE TITLE
How determine the figures for accuracy, precision in the end when use train_params['resume']

### ISSUE THREAD

**Issue by oussaifi-majdi** [Deci-AI/super-gradients#1484]
### 💡 Your Question

I use summary metrics to train the data :

train_params['resume'] = True
train_params["resume_path"] ='/content/gdrive/MyDrive/Cv/checkpointss92/yolo_nas_faces92/ckpt_latest.pth'
trainer.train(model=model,
training_params=train_params,
train_loader=train_data,
valid_loader=val_data)

but how can I determine the figures for accuracy, precision..etc with tensorboard throughout the training, from the first hours to the end

CHECKPOINT_DIR =?
EXPERIMENT_NAME =?
%load_ext tensorboard
%tensorboard --logdir {CHECKPOINT_DIR}/{EXPERIMENT_NAME} --port 6005
%reload_ext tensorboard

**Comment by oussaifi-majdi** [comment]
+1

**Comment by BloodAxe** [comment]
Sorry, I'm not sure I'm following the issue here.
Can you please elaborate what are you trying to achieve and what 'figures' are you referring to?
